src/wp-includes/media.php

```diff
@@ -1843,6 +1843,19 @@ function wp_filter_content_tags( $content, $context = null ) {
 				$filtered_image = wp_img_tag_add_loading_attr( $filtered_image, $context );
 			}
 
+			/**
+			 * Filters img tag that will be injected into the content.
+			 *
+			 * @since 6.0.0
+			 *
+			 * @param string $filtered_image  the img tag with attributes being created that will
+			 * 									replace the source img tag in the content.
+			 * @param string $context Optional. Additional context to pass to the filters.
+			 *                        Defaults to `current_filter()` when not set.
+			 * @param int    $attachment_id the ID of the image attachment.
```

Comment:
There are some problems with alignment and formatting here. Also, the parameters of the filter are not optional, so we shouldn't mention that.
```suggestion
			 * @param string $filtered_image Full img tag with attributes that will replace the source img tag.
			 * @param string $context        Additional context, like the current filter name or the function name from where this was called.
			 * @param int    $attachment_id  The image attachment ID.
```